service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/enums/MaritalStatus.java

```diff
@@ -20,13 +20,13 @@
     WIDOWED("W", "Widowed");
 
     private final String system = "http://hl7.org/fhir/v3/MaritalStatus";
-    private final String value;
+    private final String code;
     private final String display;
 
     public static uk.nhs.adaptors.oneoneone.cda.report.enums.MaritalStatus fromCode(String code) {
         return Stream.of(values())
-            .filter(am -> code.toUpperCase().equals(am.value))
+            .filter(am -> code.toUpperCase().equals(am.code))
             .findFirst()
-            .orElseThrow();
+            .orElse(null);
```

Comment:
[optional] A good modern practice is not to return null but Optional or just throw exception that value does not exist